controls/datagrid/aggregates/styling.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Aggregate Styling
+page_title: .NET MAUI DataGrid Documentation - Aggregate Styling
+description: "Review the Telerik .NET MAUI DataGrid Aggregates Styling documentation article to learn more about all built in Aggregate Styling functions you can use."
+position: 0
+slug: datagrid-aggregates-styling
+---
+
+# DataGrid Aggregates Styling
+
+The DataGrid Aggregate component provides styling mechanism customizing the look of the DataGrid Column Footer, the Group Footer and Group Header.
+
+The styling mechanism is represented by the following properties:
```

Comment:
direct action when explaining properties like this: 

Use the following properties to style the aggregates: